### ISSUE TITLE
Update example in manual

### ISSUE THREAD

**Issue by mptre** [calleerlandsson/pick#253]
I've always been bothered by the fact that I can't run one of the
examples in the manual without installing the missing utilities `fltr`
and `tac`:

```sh
$ apropos -l . | fltr | pick -do | tac | xargs man
```

Instead, I think we can come up with something as useful but portable.

Thoughts and ideas would be appreciated.

**Comment by DBOTW** [comment]
How about:
```sh
apropos -l . | sed 's/ - /\t/g' | IFS=$'\t' pick -d | awk '{print $2 " " $1}' | tr -d '()' | xargs man
```
for the above example?

**Comment by mptre** [comment]
Cool. But, I was hoping for something less complex but equally useful.
An example not related to finding manuals would be fine too.

**Comment by DBOTW** [comment]
```sh
eval "$(fc -ln 1 | pick)" # Select a line in the command history to execute
```
... works with `bash` (version 3 & 4, not 2) and `zsh` (`bash` prints some superfluous leading spaces, but I want to avoid the use of `cut` or `sed`) ... also in `README.md` it should be:
```sh
cd "$(find . -type d | pick)"
```
... to deal with spaces in directory names ...

**Comment by mptre** [comment]
> ```sh
> eval "$(fc -ln 1 | pick)" # Select a line in the command history to execute
> ```

Very nice! This should be sufficient:

```sh
$ $(fc -ln | pick)
```

> ```sh
> cd "$(find . -type d | pick)"
> ```
> ... to deal with spaces in directory names ...

Correct, care to draft a PR?

**Comment by mptre** [comment]
Does commit e50564b look good?

**Comment by DBOTW** [comment]
`eval` is evil, but by omitting the `1` after `-ln` only the last 16 entries of the history will show up ...
I will craft a PR for `README.md` ...

**Comment by mptre** [comment]
> eval is evil, but by omitting the 1 after -ln only the last 16 entries of the history will show up

Didn't know, fixed!

**Comment by DBOTW** [comment]
> eval is evil

... but `aliases` will not be expanded without `eval` and result in a `command not found`-message ...
now I remember the reason for using the evil `eval` ... ;-)

**Comment by mptre** [comment]
Thanks, fixed in commmit 031817f.

**Comment by DBOTW** [comment]
My bad: quoting the command after `eval` is not really necessary ...

**Comment by mptre** [comment]
Right, thanks. Fixed in commit ae84349.
